### ISSUE TITLE
Helper untuk kombinasi password

### ISSUE THREAD

**Issue by yaliv** [haqqi/yii2-advanced-api#21]
Bagaimana supaya memungkinkan pengguna template untuk memilih parameter untuk kombinasi password, seperti:
- huruf kecil.
- huruf besar.
- angka.
- karakter lain.

Gambarannya seperti memanggil fungsi.
```php
$rules = ['lower', 'upper', 'number', 'other'];
$password->addRules($rules);
```

termasuk juga:
- panjang minimal.
- panjang maksimal.

**Comment by haqqi** [comment]
@yaliv Setuju. Harus bisa dikonfigurasi terserah yang pakai template ini. Mungkin di config tinggal diatur "useSecurePassword"

**Comment by elfarqy** [comment]
@haqqi cuma referensi 

https://github.com/kartik-v/yii2-password

**Comment by taksetareh12** [comment]
@haqqi add

**Comment by taksetareh12** [comment]
@haqqi !help

**Comment by haqqi** [comment]
@taksetareh12 who are you? want to join this project? Seems like this project is only for internal first.

**Comment by elfarqy** [comment]
Already implemented on #31
